docs/source/nlp/usage_guides/nlp_properties_extended.rst

```diff
@@ -56,6 +78,17 @@ The model uses the roberta-base architecture, and was trained on
 `Rao and Tetreault, 2018 <https://aclanthology.org/N18-1012>`__ and online formality corpus from
 `Pavlick and Tetreault, 2016 <https://aclanthology.org/Q16-1005>`__.
 
+Examples
+~~~~~~~~
+
+================================================================  ========
```

Comment:
I assume the underline and line above should be longer than / equal to the title length. For all examples

Reply:
Not needed for correct rendering of the output